checkers/regexpPattern_checker.go

```diff
@@ -41,7 +43,16 @@ type regexpPatternChecker struct {
 	astwalk.WalkHandler
 	ctx *lintpack.CheckerContext
 
-	domainRE *regexp.Regexp
+	domainRE      *regexp.Regexp
+	beginAnchorRE *regexp.Regexp
+	endAnchorRE   *regexp.Regexp
+}
+
+type regexpFlags struct {
+	i bool // case-insensitive
+	m bool // multi-line mode: ^ and $ match begin/end line in addition to begin/end text
+	s bool // let . match \n
+	U bool // ungreedy: swap meaning of x* and x*?, x+ and x+?, etc
```

Comment:
is it ok to make it exported but leave other unexported?

Reply:
I wanted to reflect flag byte in the field name. The struct is not exported, so I don't see a problem. :)
From regexp docs:
```
i              case-insensitive (default false)
m              multi-line mode: ^ and $ match begin/end line in addition to begin/end text (default false)
s              let . match \n (default false)
U              ungreedy: swap meaning of x* and x*?, x+ and x+?, etc (default false)
```

Reply:
I mean: do we need this small (but important) inconsistency? ;D